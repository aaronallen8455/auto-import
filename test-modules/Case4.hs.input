module Case4 where

main :: IO ()
main = TIO.putStrLn $ Data.List.sort "hi"
