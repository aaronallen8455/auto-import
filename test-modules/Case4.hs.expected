module Case4 where

import qualified Data.List
import qualified Data.Text.IO as TIO

main :: IO ()
main = TIO.putStrLn $ Data.List.sort "hi"
